packages/Tabs/src/components/Panels/Panels.js

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import PropTypes from "prop-types";
+import TabsContext from "../../TabsContext";
+
+const propTypes = {
+  children: PropTypes.node.isRequired,
+};
+
+const Panels = ({ children }) => {
+  const context = React.useContext(TabsContext);
+
+  const childrenWithProps = React.Children.map(children, (panel, index) => {
+    const isSelected = context.activeIndex === index;
+
+    return React.cloneElement(panel, {
+      isSelected,
+    });
+  });
+
+  return <div>{childrenWithProps}</div>;
```

Comment:
Do we actually need this div? Or could it be a fragment?

Reply:
Can use fragment :) thanks for the suggestion!